lib/matplotlib/axes/_base.py

```diff
@@ -894,7 +908,11 @@ def set_position(self, pos, which='both'):
         self.stale = True
 
     def reset_position(self):
-        """Make the original position the active position"""
+        """
+        Reset the original position to the active position. This resets the
+        a possible position change due to aspect constraints.
+        For an explanation of the positions see `.set_position`.
+        """
```

Comment:
This is actually backwards (as was the original docstring!).  Should be 

Reset the 'active' position to the 'original' position, undoing the effect of `~.set_aspect`.

Reply:
Changed to: "Reset the active position to the original position."

It's not really undoing the effect of `set_aspect`, but of `apply_aspect`, so I don't want to adapt the second half-sentence. But explaining this is a bit too much for the docstring.

As written in #9964:
Overall, I think we would need a dedicated help page outside the docstrings to explain, coordinates, scaling and the related terminology, which is not always consistent and/or self-explanatory.

Reply:
I think users will think of it as `set_aspect`.  I also think that `set_aspect` is the sensible place to define all of this, so a link back to it would be helpful to users.  `apply_aspect` could also be mentioned.